projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.ts

```diff
@@ -347,8 +429,9 @@ export class LfFieldTemplateContainerComponent extends LfFieldContainerDirective
             await this.updateDynamicFieldsAsync(fieldInfo, values, index);
           }
         } catch (err: any) {
+          this.templateState = TemplateState.HAS_ERROR;
           this.templateErrorMessage = this.TEMPLATE_HAS_FAILED_TO_LOAD;
-          console.error('getDynamicFieldValueOptionsAsync failed: ' + err.message);
+          console.error('getDynamicFieldValueOptionsAsync failed: ' + err?.message ?? err?.title ?? '');
```

Comment:
wasn't there some kind of helper function for this?

Reply:
there is a utility function in site-office-apps

Reply:
yeah, maybe something we should move to js-utils eventually

Reply:
I will keep it here for now and make a story for js-utils